lib/response.js

```diff
@@ -127,6 +129,10 @@ function generatePayload (response) {
     return JSON.parse(this.payload)
   }
 
+  res.cookies = function parseCookies () {
+    return setCookie.parse(this)
+  }
```

Comment:
I would make this a getter instead. I would also create a new class that inherits from ServerResponse with it. (use util.inherits, not class).

Reply:
Is not the inherit already done here?

https://github.com/fastify/light-my-request/blob/7acd5444048b1bad60bca6edd163401b51e0181f/lib/response.js#L47

Gotcha for the getter